r2r/main/app.py

```diff
@@ -186,6 +186,21 @@ async def search(query: RAGQueryModel):
             logging.error(f":search: [Error](query={query}, error={str(e)})")
             raise HTTPException(status_code=500, detail=str(e))
 
+    @app.post("/rag_chatbot/")
```

Comment:
Same comment as above - I do not think this new endpoint is necessary based on my current understanding.